joomla/joomla-cms
# [4.2] Update 3rd party dependencies

### Summary of Changes
Updating various 3rd party dependencies in composer and npm to make dependabot happy. 


### Testing Instructions
*  Check out the 4.2-dev branch
*  Run npm ci
*  Run composer install
* Try to install and use Jooma
